dear_petition/petition/tests/test_utils.py

```diff
@@ -1,11 +1,14 @@
 import pytz
+import pytest
 from django.utils.timezone import make_aware, utc
 from datetime import datetime, date
 
 from dear_petition.petition import utils as pu
 
 from ..constants import DATETIME_FORMAT
 
+pytestmark = pytest.mark.django_db
```

Comment:
What does this do?

Reply:
This tells pytest that we need the test database for this test suite so we can get access to test fixtures like `petition`, which I use in my new test.